sickbeard/providers/tntvillage.py

```diff
@@ -146,7 +151,7 @@ def _doLogin(self):
 
         login_params = {'UserName': self.username,
                         'PassWord': self.password,
-                        'CookieDate': 1,
+                        'CookieDate': 0,
```

Comment:
@miigotu becouse i have no chance to make cookie work on tntvillage. I don't know how to make it work. It also happen that when sickrage start the first connection to tntvillage fail. i think that is due to old session.